src/components/common/toolbar.tsx

```diff
@@ -100,7 +106,7 @@ const Toolbar = (props: Props) => {
                 <ErrorCard fontSize={FontSize.Large} text={errorsWallet.mmLoading} icon={ErrorIcons.Metamask} />
             ) : null}
             {!isMmLocked && !isMmNotInstalled && !isMmLoading ? <ToolbarCenter>{centerContent}</ToolbarCenter> : null}
-            {!isMmLocked && !isMmNotInstalled && !isMmLoading ? <ToolbarEnd>{endContent}</ToolbarEnd> : null}
+            {!isMmLocked && !isMmNotInstalled && !isMmLoading ? <ToolbarEnd>{endContentWrapper}</ToolbarEnd> : null}
```

Comment:
Req: This code is duplicated with the line above, I think we can find a way to write it better

Reply:
It's done